File: snippets/fsharp/VS_Snippets_CLR_System/System.Console.ReadKey/fs/ReadKey1.fs
module ReadKey1

// <Snippet1>
open System

let dat = DateTime.Now
printfn $"The time: {dat:d} at {dat:t}"

let tz = TimeZoneInfo.Local
printfn $"The time zone: {if tz.IsDaylightSavingTime dat then tz.DaylightName else tz.StandardName}\n"
printf"Press <Enter> to exit... "
while Console.ReadKey().Key <> ConsoleKey.Enter do ()


// The example displays output like the following:
//     The time: 12/28/2021 at 8:35 PM
//     The time zone: Pacific Standard Time
// </Snippet1>

File: snippets/csharp/System/Console/ReadKey/ReadKey1.cs
// <Snippet1>
using System;

public class ReadKeyExample1
{
    public static void Run()
    {
        DateTime dat = DateTime.Now;
        Console.WriteLine("The time: {0:d} at {0:t}", dat);
        TimeZoneInfo tz = TimeZoneInfo.Local;
        Console.WriteLine($"The time zone: {(tz.IsDaylightSavingTime(dat) ?
                             tz.DaylightName : tz.StandardName)}\n");
        Console.Write("Press <Enter> to exit... ");
        while (Console.ReadKey().Key != ConsoleKey.Enter) { }
    }
}
// The example displays output like the following:
//     The time: 11/11/2015 at 4:02 PM:
//     The time zone: Pacific Standard Time
// </Snippet1>
